### PR TITLE
Delete monster

### LINKED ISSUE #15: delete-a-monster/ delete
As a user I want to delete a monster.

### PR DESCRIPTION
**Pull request by ElisabethSeitz**

### CODE REVIEW

File: frontend/src/service/MonsterService.js [hunk 1]
```diff
@@ -20,3 +20,13 @@ export const updateMonster = (id, userId, name, image, token) =>
   axios
     .put('/api/monster/' + id, { id, userId, name, image }, header(token))
     .then((response) => response.data);
+
+export const removeMonster = (id, userId, token) =>
+  axios({
+    method: 'DELETE',
+    url: '/api/monster/' + id,
+    data: { id, userId },
+    headers: {
```
Comment (christophersiem) [inline frontend/src/service/MonsterService.js:29]: you can use the header declared in line 3 here.

File: frontend/src/service/MonsterService.js [hunk 2]
```diff
@@ -20,3 +20,13 @@ export const updateMonster = (id, userId, name, image, token) =>
   axios
     .put('/api/monster/' + id, { id, userId, name, image }, header(token))
     .then((response) => response.data);
+
+export const removeMonster = (id, userId, token) =>
```
Comment (christophersiem) [inline frontend/src/service/MonsterService.js:24]: why did you change the writing style of this axios method (compared to get/post/put - methods)?

File: backend/src/main/java/de/neuefische/finalproject/ohboy/controller/MonsterController.java [hunk 3]
```diff
@@ -47,4 +48,12 @@ public Monster update(@RequestBody UpdateMonsterDto updatedMonster, @PathVariabl
         return monsterService.update(updatedMonster);
     }
 
+    @DeleteMapping("{monsterId}")
+    public void remove(@RequestBody RemoveMonsterDto removedMonster, @PathVariable String monsterId) {
```
Comment (christophersiem) [inline backend/src/main/java/de/neuefische/finalproject/ohboy/controller/MonsterController.java]: consider using another name instead of "removedMonster" since the monster is not yet removed at this stage

**Review by christophersiem — APPROVED** [review]
nice work 🦊

**Review by Asamya — APPROVED** [review]
The .builder declaration in unit test makes the code more readable. Good work.

File: frontend/src/service/MonsterService.js (hunk 1 above)
Comment (ElisabethSeitz) [reply frontend/src/service/MonsterService.js:29]: I tried that, but was not possible with this kind of syntax.

File: frontend/src/service/MonsterService.js (hunk 2 above)
Comment (ElisabethSeitz) [reply frontend/src/service/MonsterService.js:24]: axios.delete is not designed to send data. I have therefore decided to use this syntax.
